### PR TITLE
Add generated code and SDK API tests for Personalizer v1.0 preview

### PR DESCRIPTION
**Pull request by dwaijam**
Add generated code and SDK API tests for Personalizer v1.0 preview
Spec PR links
https://github.com/Azure/azure-rest-api-specs/pull/5743
https://github.com/Azure/azure-rest-api-specs/pull/5777
Both Spec PRs have been merged

### CODE REVIEW

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- Please do not move/edit code below this line -->
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <!-- Please do not move/edit code above this line -->
+
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.CognitiveServices.Personalizer</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Personalizer APIs.</Description>
+    <Version>1.0.0</Version>
```
Comment (yangyuan) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: maybe a preview version as the service is public preview?

**Review by yangyuan — APPROVED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj (hunk 1 above)
Comment (dsgouda) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: @dwaijam Please mark the version as `0.8.0-preview`

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Reflection;
+using System.Resources;
+
+[assembly: AssemblyTitle("Microsoft Azure Cognitive Services Personalizer Library")]
+[assembly: AssemblyDescription("Provides API functions for consuming Microsoft Azure Cognitive Services Personalizer APIs.")]
+
+[assembly: AssemblyVersion("1.0.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs]: Please set this to `0.8.0.0`

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+[assembly: AssemblyFileVersion("1.0.0.0")]
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs]: Please set this to `0.8.0.0`

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1 [hunk 4]
```diff
@@ -0,0 +1 @@
+Start-AutoRestCodeGeneration -ResourceProvider "cognitiveservices/data-plane/Personalizer" -AutoRestVersion "latest" -SdkGenerationDirectory "$PSScriptRoot\Generated\Personalizer" -ConfigFileTag "release_1_0_preview"
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1]: Is the tag `release_1_0_preview` set to default in readme.md, if yes, this is not needed

Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1]: Please generate the code using this script once the Rest spec is merged. This should create a SdkInfo_*.cs and .txt files, please commit these files

**Review by dsgouda — CHANGES_REQUESTED** [review]

**Comment by dsgouda** [comment]
Created for https://github.com/Azure/azure-rest-api-specs/pull/5743

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj (hunk 1 above)
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: done

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Properties/AssemblyInfo.cs]: done

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1 (hunk 4 above)
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1]: removed

Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/generate.ps1]: added

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    <PackageId>Microsoft.Azure.CognitiveServices.Personalizer</PackageId>
+    <Description>This client library provides access to the Microsoft Cognitive Services Personalizer APIs.</Description>
+    <Version>0.8.0-preview</Version>
+    <AssemblyName>Microsoft.Azure.CognitiveServices.Personalizer</AssemblyName>
+    <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;Personalizer API;REST HTTP client</PackageTags>
+    <PackageReleaseNotes>
+    <![CDATA[
+    Microsoft Cognitive Services Personalizer SDK.
+
+    This package calls the API version "v0.8.0-preview"
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: I am not sure that is correct. Please check what REST API version this corresponds to

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj (hunk 5 above)
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Personalizer.csproj]: fixed

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer.Tests.csproj [hunk 6]
```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.Personalizer.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Personalizer.Tests</AssemblyName>
-    <Version>1.0.0</Version>
+    <Version>0.8.0-preview</Version>
```
Comment (dsgouda) [inline src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer.Tests.csproj]: Test project version must be set to 1.0.0 and never modified. Not blocking but must be fixed

**Review by dsgouda — APPROVED** [review]
LGTM will merge once CIs pass

File: src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer.Tests.csproj (hunk 6 above)
Comment (dwaijam) [reply src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer.Tests.csproj]: fixed

**Pull request merged**